tests/basic/pos/ReflExt03A.hs

```diff
@@ -0,0 +1,15 @@
+-- | test reflection of private variables of imported modules
+
+{-@ LIQUID "--ple"         @-}
+{-@ LIQUID "--reflection" @-}
+
+module ReflExt03A where
+
+import ReflExt03B
+
+{-@ reflect f @-}
+{-@ reflect ReflExt03B.myAdd @-}
```

Comment:
It would be helpful to also have a test that checks that opaque reflection works when unfolding from interface files.

Reply:
Done in a2bd4dd698e671a1b041909d20bc133cd73089c6